guides/svg/template.md

```diff
@@ -0,0 +1,229 @@
+# How to build a component model
+
+## Example of model :
+
+For this example, we use a `container component`. It contains all necessary information to understand how to transform
+it into a "non-container" component.
+
+### A basic model of container component :
+
+```jsx
+<svg class="template" width="250" height="200">
+    <rect class="component-hitbox" width="100%" height="100%" fill="white" />
+
+    <g transform="translate(5,5)">
+        <svg id="icon-{{id}}" class="component-icon" width="32" height="32"></svg>
+    
+        <g transform="translate(35,0)">
+            <text class="component-name" y="1em">{{ name }}</text>
+            <text class="component-type" y="2.5em">{{ definition.type }}</text>
+        </g>
+    </g>
+
+    <g transform="translate(0,50)">
+        <svg class="component-container" overflow="visible">
+            <rect class="container-background" width="100%" height="100%" fill="whitesmoke" />
+        </svg>
+    </g>
+    
+    {% if hasError %}
+    <g class="component-error">
+        <svg width="15" height="15" x="200" y="10" viewBox="0 0 100 100">
+          <circle fill="red" r="50" cx="50" cy="50"></circle>
+        </svg>
+    </g>
+    {% endif %}
+</svg>
+```
+
+### Why I see TWIG syntax in this model ?
+
+As you can see, we have some `twig` syntax in the template svg, **but** that's **not** `twig`. To render the template,
```

Comment:
add list of variable / function we can use in it

Reply:
Done

Reply:
Miss `hasError` and `getAttribute` function

Reply:
Done